### PR TITLE
feat: import dasboard changes for UI de-clutter

### PR DESCRIPTION
**Pull request by Ian2012**
### Description

This PR imports the changes for the dashboards for the UI de-clutter epic.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @Ian2012!

### What's next?

*Please work through the following steps to get your changes ready for engineering review:*

#### :radio_button: Get product approval

If you haven't already, [check this list](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Does-my-contribution-require-Product-Review%3F) to see if your contribution needs to go through the product review process.

- If it does, you'll need to submit a product proposal for your contribution, and have it reviewed by the [Product Working Group](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3449028609/Product+Working+Group).
    - This process (including the steps you'll need to take) is documented [here](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Product-Review-Process).
- If it doesn't, simply proceed with the next step.

#### :radio_button: Provide context

To help your reviewers and other members of the community understand the purpose and larger context of your changes, feel free to add as much of the following information to the PR description as you can:

- Dependencies
  > This PR must be merged before / after / at the same time as ...
- Blockers
  > This PR is waiting for OEP-1234 to be accepted.
- Timeline information
  > This PR must be merged by XX date **because** ...
- Partner information
  > This is for a course on edx.org.
- Supporting documentation
- Relevant [Open edX discussion forum](https://discuss.openedx.org/) threads





#### :radio_button: Get a green build

If one or more checks are failing, continue working on your changes until this is no longer the case and your build turns green.



#### :radio_button: Let us know that your PR is ready for review:

### Who will review my changes?


This repository is currently maintained by `@bmtcril`.  Tag them in a comment and let them know that your changes are ready for review.


### Where can I find more information?

If you'd like to get more details on all aspects of the review process for open source pull requests (OSPRs), check out the following resources:

- [Overview of Review Process for Community Contributions](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/FAQ-about-pull-requests.html)
- [Pull Request Status Guide](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/pull-request-statuses.html)
- [Making changes to your pull request](https://docs.openedx.org/en/latest/documentors/how-tos/make_changes_to_your_pull_request.html)

### When can I expect my changes to be merged?

Our goal is to get community contributions seen and reviewed as efficiently as possible.

However, the amount of time that it takes to review and merge a PR can vary significantly based on factors such as:

- The size and impact of the changes that it introduces
- The need for product review
- Maintenance status of the parent repository

:bulb: *As a result it may take up to several weeks or months to complete a review and merge your PR.*

File: tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/charts/Number_of_Views_across_Video_Duration_a7947b.yaml [hunk 1]
```diff
@@ -1,9 +1,13 @@
-_file_name: Watched_Video_Segments_a7947b.yaml
+_file_name: Number_of_Views_across_Video_Duration_a7947b.yaml
 cache_timeout: null
 certification_details: null
 certified_by: null
 dataset_uuid: c2f0e842-738d-41a0-9ea6-51e32b7fab82
-description: null
+description: "Key result 1\tDefinition and Implementation of Hiring Model for Foreign\
```
Comment (Ian2012) [inline tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/charts/Number_of_Views_across_Video_Duration_a7947b.yaml:6]: I'm not sure were did this come from hahaha

**Comment by bmtcril** [comment]
Issues I'm seeing when running this PR:

All dashboards:
- Video Engagement per Section/Subsection is always empty across all courses

Individual Learner:
- In the problems sub-tab instead of "Problem Attempts" and "Problem Results" charts, I get two "Problem Results" charts and they're both always empty

**Review by bmtcril — APPROVED** [review]
I am seeing some odd things locally, but we can merge this and see how it looks on a real sandbox as long as things are looking good before the next release.

**Comment by Ian2012** [comment]
I will fix all of that on the new environment

**Pull request merged**